src/utils/collision.js

```diff
@@ -4,13 +4,6 @@ const SEPARATING_LINE_FOUND = 'separatingLineFound';
 const SEPARATING_LINE_NOT_FOUND = 'separatingLineNotFound';
 
 export function triangleCollides (triangle, viewportAABB) {
-    const viewport = [
-        { x: viewportAABB.minx, y: viewportAABB.miny },
-        { x: viewportAABB.minx, y: viewportAABB.maxy },
-        { x: viewportAABB.maxx, y: viewportAABB.miny },
-        { x: viewportAABB.maxx, y: viewportAABB.maxy }
-    ];
-
     /*
      * TODO
```

Comment:
I don't think so, this was done some time ago to remark some possible ways to optimize the function. Since it is a performance intensive function, we should probably review this in the future.